src/styles/_page.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  margin: 0;
+
+  font: {
+    family: $font;
+    size: $font-size;
+  }
```

Comment:
The use of nested properties for the 'font' shorthand is incorrect. According to CSS syntax, 'font' should be a single shorthand property, not a nested block. You should use 'font-family' and 'font-size' separately or use the 'font' shorthand correctly.